project/YuChiehLiu/README.md

```diff
@@ -0,0 +1,54 @@
+## Project Name
+
+Implementing Optimal Binary Search Tree(OBST) with C++ Program
+
+
+## Basic Information
+
+OBST is that given a sorted array key[0.. n-1] of search keys and an array freq[0.. n-1] of frequency counts, where freq[i] is the number of searches for keys[i]. And construct a binary search tree of all keys such that the total cost of all the searches is as small as possible.

```

Comment:
This looks more like a project for pure C++ use.  Could you analyze how it may be used or behave by comparing with `std::map`?